docs/installing/manual_install.md

```diff
@@ -411,6 +411,13 @@ site in action.
 
 You can daemonize the process by starting it with the `--daemonize` option.
 
+
+If you anticipate getting a lot of traffic, you can configure thin to run as a cluster
```

Comment:
This section can then be moved to Generate application daemon/THIN/Template Variables